test/cljc/reitit/ring_coercion_test.cljc

```diff
@@ -606,53 +606,70 @@
-                                                   :body default-response}}
-                                  :handler (fn [req]
-                                             {:status 200
-                                              :body (-> req :parameters :request)})}}]
-                  {#_#_:validate reitit.ring.spec/validate
-                   :data {:middleware [rrc/coerce-request-middleware
-                                       rrc/coerce-response-middleware]
-                          :coercion coercion}}))
-            call (fn [request]
-                   (try
-                     (app request)
-                     (catch ExceptionInfo e
-                       (select-keys (ex-data e) [:type :in]))))
-            request (fn [request-format response-format body]
-                      {:request-method :post
-                       :uri "/foo"
-                       :muuntaja/request {:format request-format}
-                       :muuntaja/response {:format response-format}
-                       :body-params body})]
-        (testing "succesful call"
-          (is (= {:status 200 :body {:request :json, :response :json}}
-                 (call (request "application/json" "application/json" {:request :json :response :json}))))
-          (is (= {:status 200 :body {:request :edn, :response :json}}
-                 (call (request "application/edn" "application/json" {:request :edn :response :json}))))
-          (is (= {:status 200 :body {:request :default, :response :default}}
-                 (call (request "application/transit" "application/transit" {:request :default :response :default})))))
-        (testing "request validation fails"
-          (is (= {:type :reitit.coercion/request-coercion :in [:request :body-params]}
-                 (call (request "application/edn" "application/json" {:request :json :response :json}))))
-          (is (= {:type :reitit.coercion/request-coercion :in [:request :body-params]}
-                 (call (request "application/json" "application/json" {:request :edn :response :json}))))
-          (is (= {:type :reitit.coercion/request-coercion :in [:request :body-params]}
-                 (call (request "application/transit" "application/json" {:request :edn :response :json})))))
-        (testing "response validation fails"
-          (is (= {:type :reitit.coercion/response-coercion :in [:response :body]}
-                 (call (request "application/json" "application/json" {:request :json :response :edn}))))
-          (is (= {:type :reitit.coercion/response-coercion :in [:response :body]}
-                 (call (request "application/json" "application/edn" {:request :json :response :json}))))
-          (is (= {:type :reitit.coercion/response-coercion :in [:response :body]}
-                 (call (request "application/json" "application/transit" {:request :json :response :json})))))))))
+      (doseq [app [(ring/ring-handler
```

Comment:
you should name the test cases using `(testing ...)`, otherwise it's impossible to know from the test report which case failed

Reply:
good point, fixed. btw, Cursive breaks on `(testing <<non-string>> ...)`, so made them all strings